OracleManagementAgent/dockerfiles/latest/user-scripts/init-agent.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-# Copyright (c) 2022 Oracle and/or its affiliates.
+# Copyright (c) 2023 Oracle and/or its affiliates.
```

Comment:
I'm not sure you can claim Oracle copyright on a script that contains nothing of value and will be populated by the user